bindings/python/README.md

```diff
@@ -954,18 +871,17 @@ account_balance = {
 }
 ```
 
-### ClientOptions
+## ClientOptions
 
 A dict with the following key:value pairs.
 
 ```python
 client_options = {
-    'primary_node': [Node](#node)] (optional),
-    'primary_pow_node': [Node](#node)] (optional),
     'nodes': list[[Node](#node)] (optional),
+    'primary_node': [Node](#node)] (optional), 
+    'primary_pow_node': [Node](#node)] (optional),
     'node_pool_urls': list[str] (optional),
     'network': str (optional),
-    'mqtt_enabled': bool (optional),
```

Comment:
This should stay

Reply:
Thanks for catching that! Yes i need to modify the api_reference.md too.